### PR TITLE
Unconditional img generation - about.md file

### PR DESCRIPTION
**Pull request by juancopi81**
PR to add the about.md file with the task description. @congaco is working on the other items of this file :).

### CODE REVIEW

File: tasks/src/unconditional-image-generation/about.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## About the Task
+
+Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.
```
Comment (merveenoyan) [inline tasks/src/unconditional-image-generation/about.md:3]:
```suggestion
Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.
```
No need to write this given it's also on `data.ts` no?

File: tasks/src/unconditional-image-generation/about.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Unconditional image generation models usually start with a *seed* that generates a *random noise vector*. The model will then upsample this vector to create the final image. 
+An example of unconditional image generation would be generating the image of a face on a model trained with the [CelebA dataset](https://huggingface.co/datasets/huggan/CelebA-HQ).
+
+## Use Cases
```
Comment (merveenoyan) [inline tasks/src/unconditional-image-generation/about.md:7]: Can you leave these sections descriptions as well so it's easier for others to contribute🙂

File: tasks/src/unconditional-image-generation/about.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## About the Task
+
+Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.
+Unconditional image generation models usually start with a *seed* that generates a *random noise vector*. The model will then upsample this vector to create the final image.
```
Comment (merveenoyan) [inline tasks/src/unconditional-image-generation/about.md:4]: Can you explain upsampling a little 😅

File: tasks/src/unconditional-image-generation/about.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+## About the Task
+
+Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.
+Unconditional image generation models usually start with a *seed* that generates a *random noise vector*. The model will then upsample this vector to create the final image. 
+An example of unconditional image generation would be generating the image of a face on a model trained with the [CelebA dataset](https://huggingface.co/datasets/huggan/CelebA-HQ).
```
Comment (merveenoyan) [inline tasks/src/unconditional-image-generation/about.md:5]:
```suggestion
An example of unconditional image generation would be generating an image of a face on a model trained with the [CelebA dataset](https://huggingface.co/datasets/huggan/CelebA-HQ).
```

**Review by merveenoyan — COMMENTED** [review]
Overall looks good to me, happy to merge after comments are fixed 🙂

File: tasks/src/unconditional-image-generation/about.md (hunk 1 above)
Comment (juancopi81) [reply tasks/src/unconditional-image-generation/about.md:3]: Oh sure! :)

**Comment by juancopi81** [comment]
Hey @merveenoyan I added some changes based on your comments. I am leaving some description in the other parts of the aboud.md file so @congaco can also contribute. I am happy to add also those parts if he is not longer available! Thank you very much.

**Comment by merveenoyan** [comment]
@congaco you don't have to wait! You can add below sections in a separate PR of yours:
```
## Use Cases

You can contribute this area with common use cases of the task!

## Model Hosting and Inference

This section should have useful information about Model Hosting and Inference

## Useful Resources

In this area, you can insert useful resources about how to train or use a model for this task.

### Training your own model in just a few seconds

In this area, you can insert useful information about training the model
```

File: tasks/src/unconditional-image-generation/about.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+## About the Task
+
+Unconditional image generation models usually start with a *seed* that generates a *random noise vector*. The model will then use this vector to create an output image similar to the images used for training the model. 
+
+An example of unconditional image generation would be generating the image of a face on a model trained with the [CelebA dataset](https://huggingface.co/datasets/huggan/CelebA-HQ) or [generating a butterfly](https://huggingface.co/spaces/huggan/butterfly-gan) on a model trained with the [Smithsonian Butterflies dataset](https://huggingface.co/datasets/ceyda/smithsonian_butterflies).
+
+[GAN](https://en.wikipedia.org/wiki/Generative_adversarial_network) and [Diffusion](https://huggingface.co/docs/diffusers/index) are common models for this task.
```
Comment (merveenoyan) [inline tasks/src/unconditional-image-generation/about.md:7]:
```suggestion
[Generative adversarial networks](https://en.wikipedia.org/wiki/Generative_adversarial_network) and [Diffusion](https://huggingface.co/docs/diffusers/index) are common architectures for this task.
```

**Review by merveenoyan — APPROVED** [review]
Thanks a lot!

**Comment by merveenoyan** [comment]
@juancopi81 what is your HF username? 🙂

**Comment by juancopi81** [comment]
Hi @merveenoyan thanks a lot! My HF username is juancopi81  I noticed @congaco did not finish the other items, maybe he is busy? I am happy to finish the document so it is not incomplete.... Please let me know :)

**Pull request merged**